crates/bevy_render/src/camera/bundle.rs

```diff
@@ -1,36 +1,48 @@
 use crate::{
-    camera::{
-        Camera, CameraPlugin, DepthCalculation, OrthographicProjection, PerspectiveProjection,
-        ScalingMode,
-    },
+    camera::{Camera, DepthCalculation, OrthographicProjection, PerspectiveProjection},
     primitives::Frustum,
     view::VisibleEntities,
 };
-use bevy_ecs::bundle::Bundle;
+use bevy_ecs::{bundle::Bundle, prelude::Component};
 use bevy_math::Vec3;
 use bevy_transform::components::{GlobalTransform, Transform};
 
-use super::CameraProjection;
+use super::{CameraProjection, ScalingMode};
+
+#[derive(Component, Default)]
+pub struct Camera3d;
+
+#[derive(Component, Default)]
+pub struct Camera2d;
 
 /// Component bundle for camera entities with perspective projection
 ///
 /// Use this for 3D rendering.
 #[derive(Bundle)]
-pub struct PerspectiveCameraBundle {
+pub struct PerspectiveCameraBundle<M: Component> {
     pub camera: Camera,
     pub perspective_projection: PerspectiveProjection,
     pub visible_entities: VisibleEntities,
     pub frustum: Frustum,
     pub transform: Transform,
     pub global_transform: GlobalTransform,
+    pub marker: M,
+}
+
+impl Default for PerspectiveCameraBundle<Camera3d> {
+    fn default() -> Self {
+        PerspectiveCameraBundle::new_3d()
+    }
 }
 
-impl PerspectiveCameraBundle {
+impl PerspectiveCameraBundle<Camera3d> {
     pub fn new_3d() -> Self {
-        Default::default()
+        PerspectiveCameraBundle::new()
```

Comment:
This feels a bit weird to me? Why not just use `PerspectiveCameraBundle<Camera3d>::default()` instead? And then specialize the default impls based on which generic is passed in.